content/en/docs/components/training/reference/architecture.md

```diff
@@ -0,0 +1,33 @@
++++
```

Comment:
@tenzen-y @andreyvelich please feel free to review as I did my best to incorporate the notes from the doc. I also mentioned this was V1. In the future we could add V1 to the title once V2 is ready.